xml_converter/src/attribute/marker_category.cpp

```diff
@@ -0,0 +1,13 @@
+#include "marker_category.hpp"
+
+#include <string>
+#include <vector>
+
+#include "../rapid_helpers.hpp"
+#include "../rapidxml-1.13/rapidxml.hpp"
+
+MarkerCategory parse_marker_category(rapidxml::xml_attribute<>* input, vector<XMLError*> *) {
+    MarkerCategory marker_category;
+    marker_category.category = get_attribute_value(input);
```

Comment:
We should validate that this is a real marker category and if it is not then create an error.